models/migrations/v103.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"crypto/md5"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"code.gitea.io/gitea/modules/setting"
+
+	"xorm.io/xorm"
+)
+
+func renameExistingUserAvatarName(x *xorm.Engine) error {
+	sess := x.NewSession()
+	defer sess.Close()
+	if err := sess.Begin(); err != nil {
+		return err
+	}
+
+	type User struct {
+		ID     int64 `xorm:"pk autoincr"`
+		Avatar string
+	}
+
+	users := make([]User, 0)
+	if err := sess.Find(&users); err != nil {
```

Comment:
Assume you have thousands of users?

Reply:
Yeah this this should be done in in batches, see other migrations

Reply:
@lunny , @lafriks I processed them in batches. Do they look okay ?